code/__HELPERS/see_through_maps.dm

```diff
@@ -10,6 +10,10 @@
 #define SEE_THROUGH_MAP_THREE_X_TWO "3x2"
 ///One row of three tiles above the atom, but offset one tile to the left because of how billboards work
 #define SEE_THROUGH_MAP_BILLBOARD "billboard"
+///Two rows of three wide, but offset one tile to the left because thats how shipping containers work
+#define SEE_THROUGH_MAP_SHIPPING_CONTAINER "shipping_container"
+///I'm not even gonna try and describe it
```

Comment:
You do have to document this, you don't need to describe exactly what it is you just need to describe where it's used